src/main/java/angela/command/GetTaskFromDateCommand.java

```diff
@@ -0,0 +1,48 @@
+package angela.command;
+
+import angela.datetime.DateTable;
+import angela.task.TaskList;
+import angela.util.BotStorage;
+import angela.util.Ui;
+
+/**
+ * Extracts events on a specific date command
+ */
+public class GetTaskFromDateCommand extends angela.command.Command {
```

Comment:
I'm not sure whether I like the name or not. The name feels like a method rather than a class. Perhaps you could reconsider?

Reply:
This is also true for several of the commands classes.